app.js

```diff
@@ -1,11 +1,13 @@
 const express = require('express');
 const app = express();
-const {getTopics} = require('./controllers/ncnews.controller');
+const {getTopics, getEndpoints} = require('./controllers/ncnews.controller');
 
 app.use(express.json());
```

Comment:
We don't need express.json() until we build a post or a path so please remove :)